CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
     continue to call `generateForAnnotatedElement` repeatedly for each element.
   * `GeneratorForAnnotation` passes in a `ConstantReader` for the annotation
     instance rather than re-creating it using mirrors.
+  * `GeneratorBuilder` is replaced with `PartBuilder` and `LibraryBuilder`
+    depending on whether the `Generator`(s) used are meant to be included in a
```

Comment:
Tough to parse this.

`depending on...` -> `depending on whether the target is a part file or a stand-alone library.` Maybe?